lib/nerves_init_gadget/network_manager.ex

```diff
@@ -19,7 +19,7 @@ defmodule Nerves.InitGadget.NetworkManager do
     SystemRegistry.register()
 
     # Initialize networking
-    Nerves.Network.setup(opts.ifname, ipv4_address_method: opts.address_method)
+    Nerves.Network.setup(opts.ifname)
```

Comment:
This allows people to set their `nerves_network` options in the `defaults` config section, which I think resolves #24. Simply deleting it here does eliminate the option to configure `address_method` in the `nerves_init_gadget` config, but I'm not sure whether we should put some code in here to merge it, or just remove that option because it's somewhat confusing to have yet another place you can set that option.